src/main/target/OMNIBUSF4FW/OMNIBUSF4V6.mk

```diff
@@ -0,0 +1 @@
+#OMNIBUSF4V6
```

Comment:
Looks like your rebase went wrong. :-(

Reply:
Hmm, that's interesting because I checked pulled master and branched off it.  I'll check...